packages/terra-application-navigation/tests/jest/extensions/Extensions.test.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Extensions from '../../../src/extensions/_Extensions';
+import mockIntl from '../mockIntl';
```

Comment:
mockIntl is provided in the terra-enzyme-intl package: https://github.com/cerner/terra-enzyme-intl#mockintl

Reply:
Updated.